Source/Core/Core/Boot/Boot.cpp

```diff
@@ -300,19 +308,46 @@ bool CBoot::LoadMapFromFilename()
   return false;
 }
 
+static void RAMOverrideForIPLMemoryValues(std::string& data, const u32 ipl_hash)
+{
+  if (!Config::Get(Config::MAIN_RAM_OVERRIDE_ENABLE))
+    return;
+
+  // Pre-calculate the new values.
+  const s16 mem1_s_size = Common::swap16(Config::Get(Config::MAIN_MEM1_SIZE) >> 16);
+  const u32 nop = Common::swap32(0x60000000);
+
+  // [lis	r0, 0x0180] <-- This instruction is what sets the Simulated MEM1 Size in retail IPLs.
```

Comment:
As far as I can tell, it's writing to `80000028`, which [wiibrew calls](https://wiibrew.org/wiki/Memory_Map) is "Memory Size (Physical)", not Simulated MEM1 size.  Simulated Memory Size is 800000F0 instead (which doesn't appear to be written by the IPL).

Reply:
My mistake, I just assumed it was simulated.